root.json

```
@@ -61,6 +62,7 @@
     "Rocket.Chat": "rocketchat.json",
     "sabnzb": "sabnzb.json",
     "Seafile": "Seafile.json",
+    "SickChill": "sickchill.json,
```

Missing closing quote:
```suggestion
    "SickChill": "sickchill.json",
```